src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -1,40 +1,46 @@
 package frc.robot.subsystems;
 
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import javax.naming.ldap.Control;
+
 import frc.robot.Config;
 import frc.robot.Logger;
 
 
 public class Climber extends Subsystem {
+  private final int PORT_NUMBER = Config.getInstance().getInt(Config.Key.CLIMBER__MOTOR);
 
-  //TODO: Tune ALL of these values
-  /** Speed of motor in % of maximum output*/
-  private final double EXTEND_SPEED = Config.getInstance().getDouble(Config.Key.CLIMBER__EXTEND_SPEED);
-  private final double RETRACT_SPEED = Config.getInstance().getDouble(Config.Key.CLIMBER__RETRACT_SPEED);
+  //TODO: Tune these values
+  private final double HEIGHT_IN_ENCODER_TICKS = Config.getInstance().getDouble(Config.Key.CLIMBER__HEIGHT_IN_ENCODER_TICKS);
+  private final double RETRACTED_HEIGHT_IN_ENCODER_TICKS = Config.getInstance().getDouble(Config.Key.CLIMBER__RETRACTED_HEIGHT_IN_ENCODER_TICKS);
+  private final double HOMING_SPEED_PERCENT = -.2d;
```

Comment:
The `d` is redundant here, since you're declaring the variable as a double.

Reply:
Removed